tree/treeplayer/inc/TSimpleAnalysis.h

```diff
@@ -0,0 +1,81 @@
+// @(#)root/tree:$Id$
+// Author: Luca Giommi   22/08/16
+
+/*************************************************************************
+ * Copyright (C) 1995-2000, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT_TSimpleAnalysis
+#define ROOT_TSimpleAnalysis
+
+//////////////////////////////////////////////////////////////////////////
+//                                                                      //
+// TSimpleAnalysis                                                      //
+//                                                                      //
+// A TSimpleAnalysis object permit to, given an input file or through   //
+// command line, create a .root file in which are saved histograms      //
```

Comment:
TSimpleAnalysis doesn't have a command line :-) I'd remove that part.

Can you describe the syntax of the config file (e.g. through an example)?